plugins/module_utils/_global_catalog.py

```diff
@@ -120,12 +119,27 @@ def _run_dfhrmutl(
 
 
 def _execute_dfhrmutl(location, sdfhload, cmd=""):   # type: (str, str, str) -> MVSCmdResponse
-    return MVSCmd.execute(
+    sysprint = OutputDefinition(record_length=133)
+
+    dds = [
+        DDStatement('steplib', DatasetDefinition(sdfhload)),
+        DDStatement('dfhgcd', DatasetDefinition(location)),
+        DDStatement('sysin', InputDefinition(content=cmd)),
+        DDStatement('sysprint', sysprint)
+    ]
+
+    response = MVSCmd.execute(
         pgm="DFHRMUTL",
-        dds=_get_rmutl_dds(location=location, sdfhload=sdfhload, cmd=cmd),
+        dds=dds,
         verbose=True,
         debug=False)
 
+    response.stdout = datasets.read(sysprint.name)
+
+    datasets.delete(sysprint.name)
+
```

Comment:
Seems unlikely to me, but if datasets.read raised a ZOAUException here I guess we might not delete the dataset. So could dtail ever throw here? Would it be better with a simple try catch to raise a MVSExecutionException so we fail the module properly?